packages/server/database/migrations/20211025134235-estimateStageTaskIdBackfill.ts

```diff
@@ -1,153 +0,0 @@
-import JiraIssueId from 'parabol-client/shared/gqlIds/JiraIssueId'
```

Comment:
i don't think you meant to delete a migration. possible bad merge?

Reply:
That's right! I pushed the funky merge just as you started reviewing. I should have moved it to Reviewer Review before pushing. All fixed now!